### ISSUE TITLE
Log level being ignored and example in docs broken

### ISSUE THREAD

**Issue by manoadamro** [pallets/flask#2788]
Using the example in the docs:

```python
import flask
import logging.config

logging.config.dictConfig({
    'version': 1,
    'formatters': {'default': {
        'format': '[%(asctime)s] %(levelname)s in %(module)s: %(message)s',
    }},
    'handlers': {'wsgi': {
        'class': 'logging.StreamHandler',
        'stream': 'ext://flask.logging.wsgi_errors_stream',
        'formatter': 'default'
    }},
    'root': {
        'level': 'INFO',
        'handlers': ['wsgi']
    }
})

app = flask.Flask(__name__)
```

produces the following error:
```
ValueError: Unable to configure handler 'wsgi': Cannot resolve 'flask.logging.wsgi_errors_stream': No module named 'flask.logging.wsgi_errors_stream'; 'flask.logging' is not a package
```

changing `ext://flask.logging.wsgi_errors_stream` to `stdout` stops the error, but log level is still ignored.

Using the following:
```python
import flask
import logging.config

logging.config.dictConfig({
    'version': 1,
    'formatters': {
        'default': {
            'format': '[%(asctime)s] %(levelname)s in %(module)s: %(message)s',
        }
    },
    'handlers': {
        'wsgi': {
            'class': 'logging.StreamHandler',
            'stream': 'stdout',
            'formatter': 'default',
        }
    },
    'root': {
        'level': 'DEBUG',
        'handlers': ['wsgi']
    }
})


app = flask.Flask(__name__)

app.logger.debug('I WORK')
app.logger.info('I WORK')
app.logger.warning('I WORK')
app.logger.error('I WORK')
app.logger.critical('I WORK')
```
(Note the log level set to `DEBUG`)

Actual Output:
```
[2018-05-17 11:35:20,069] ERROR in __init__: I WORK
[2018-05-17 11:35:20,069] CRITICAL in __init__: I WORK
```

Expected output:
```
[2018-05-17 11:35:20,069] DEBUG in __init__: I WORK
[2018-05-17 11:35:20,069] INFO in __init__: I WORK
[2018-05-17 11:35:20,069] WARNING in __init__: I WORK
[2018-05-17 11:35:20,069] ERROR in __init__: I WORK
[2018-05-17 11:35:20,069] CRITICAL in __init__: I WORK
```
I have also tried both setting the level in the handler and formatter and calling `app.logger.setLevel('DEBUG')`
neither made any difference.

TL;DR:
 - logging example in docs errors.
 - unable to set log level
---

### Environment

* Python version: 3.6.4
* Flask version: 1.0.2
* Werkzeug version: 0.14.1

**Comment by nick-otter** [comment]
+1

**Comment by davidism** [comment]
I can't reproduce your issue. Are you sure that you're running with Flask 1.0? Prior to that `wsgi_errors_stream` didn't exist in the same place. You do have an error in your second `dictConfig` where the stream should be [`ext://sys.stdout`](https://docs.python.org/3/library/logging.config.html#access-to-external-objects), but fixing that I get all five lines output.

**Comment by manoadamro** [comment]
@davidism running in a virtual env 

```
(env) pip freeze
click==6.7
Flask==1.0.2
itsdangerous==0.24
Jinja2==2.10
MarkupSafe==1.0
Werkzeug==0.14.1
```

**Comment by manoadamro** [comment]
@davidism As it turns out, the problem seems to be with pycharm. 
It's telling me its using the environment but clearly, it isn't. 

Using nano and terminal i can see it does now work as expected.
thanks and sorry for wasting your time.

**Comment by davidism** [comment]
I checked the whole thing in PyCharm from the start, and had no issues. Perhaps something's misconfigured in PyCharm related to the terminal or interpreter?

**Comment by manoadamro** [comment]
yeah, the project interpreter is set /myProject/env/bin/python3 (venv for current project), so is my terminal however pycharm is lying to me, its actually using my global interpreter, because it shows a totally different list when i call pip freeze. either way, not you problem, 
thanks again.

**Comment by ye** [comment]
@davidism on on May 17, 2018 you said

> You do have an error in your second dictConfig where the stream should be ext://sys.stdout

Should `ext://sys.stdout` be the value for dictConfig's `handlers.wsgi.stream`? If so, why [Flask 1.1.x official doc](https://flask.palletsprojects.com/en/1.1.x/logging/) still reference `ext://flask.logging.wsgi_errors_stream` in the example? If not, what's the difference? I am confused.
